pkg/gui/gui.go

```diff
@@ -535,7 +535,11 @@ func NewGui(
 	)
 
 	osCommand := oscommands.NewOSCommand(cmn, config, oscommands.GetPlatform(), guiIO)
-
+	shell := gui.UserConfig.OS.Shell
+	if shell != "" {
+		osCommand.Platform.Shell = shell
+		osCommand.Platform.ShellArg = gui.UserConfig.OS.ShellArg
+	}
```

Comment:
Sounds good. I don't know how to access config.OS from there, though :confused:

Reply:
Sounds good. I don't know how to access config.OS from there, though :confused:

Reply:
On second thought, I think it would make more sense to pass the `OSConfig` struct into `oscommands.GetPlatform` and get that function to extract the `shell` and `shellArg` values from there and use them there directly.

> Sounds good. I don't know how to access config.OS from there, though 😕

You would be able to access the config through the `Common` struct that's passed into `NewOsCommand` function.